tests/e2e/e2e_gov_test.go

```diff
@@ -176,6 +177,35 @@ func (s *IntegrationTestSuite) testGovParamChange() {
 	})
 }
 
+func (s *IntegrationTestSuite) testGovConstitutionAmendment() {
+	s.Run("constitution amendment", func() {
+		chainAAPIEndpoint := fmt.Sprintf("http://%s", s.valResources[s.chainA.id][0].GetHostPort("1317/tcp"))
+		senderAddress, _ := s.chainA.validators[0].keyInfo.GetAddress()
+		sender := senderAddress.String()
+
+		res := s.queryConstitution(chainAAPIEndpoint)
+		newConstitution := "New test constitution"
+		amendment, err := govtypes.GenerateUnifiedDiff(res.Constitution, newConstitution)
```

Comment:
Even though it is already unit-tested, for end2end completeness it would be nice to use the CLI to generate the proposal file instead, wdyt?

Reply:
sounds good!

Reply:
done